[SYCLomatic][DOC] Add more manual fix examples in the document (#1230)
oneapi-src/SYCLomatic: docs/dev_guide/diagnostic_ref/dpct1058.rst

If you want this macro to be migrated by |tool_name|, please
use the macro that needs migration in your code and pass the code to the tool
again. Otherwise you can update the code manually or ignore the warning.

For example, this original CUDA\* code:

.. code-block:: cpp
   :linenos:

>    #include <cstdio>

Reviewer: ```suggestion
  

```

Author: done

   
   #define ATOMIC_ADD(x, v)  atomicAdd(&x, v);
   
   void foo() {

>      // User's code which is not related to ATOMIC_ADD

Reviewer: ```suggestion
     // There is no using of ATOMIC_ADD macro

```

Author: done
